### DISCUSSION TITLE
Localization

### DISCUSSION THREAD

**Discussion by ghost** [tokio-rs/axum#2700] [Q&A]
### Summary

Hi,

I'm looking for guidance/examples on how to properly implement localization with Axum but haven't found anything.  Any tips?

Thanks,

### axum version

0.7.5

**Comment by nikola-bozin-org** [comment]
@tristanguerragomez you can check this [crate](https://crates.io/crates/cargo-i18n), it can help you with localization.

**Reply by ghost** [reply]
Thanks for pointing that out @nikola-bozin-org.  Lots of ways of going about this... trying to find a simple yet effective way to go about this, but it's not easy, very fragmented ecosystem.
